users.html

```diff
@@ -483,7 +516,7 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   L.marker([22.840991688686348, 108.32017390098716], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Nanning Normal University</strong><br>Nanning, Guangxi, China<ul><li><a href='https://www.nnnu.edu.cn/' target='_new'>Website</a></li></ul>");
 
   // Nanyang Technological University
-  L.marker([1.3483313503715026, 103.68304886562292], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Nanyang Technological University</strong><br>Singapore<ul><li><a href='https://www.ntu.edu.sg/' target='_new'>Website</a></li></ul>");
+  L.marker([1.3483313503715026, 103.68304886562292], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Nanyang Technological University</strong><br>Singapore<ul><li><a href='https://earthobservatory.sg/research/climate/air-pollution-and-health' target='_new'>Earth Observatory of Singapore</a></li></ul>");
```

Comment:
Question: The link is Earth Observatory of Singapore but the label is Nanyang University.  Just wanted to make sure they are affliliated.